### PR TITLE
Fix/4/link domain name

### LINKED ISSUE #4: [Domains] Link a domain name to an application

### PR DESCRIPTION
**Pull request by ichbinkour**
Fix #4

### CODE REVIEW

File: src/Domains/index.js [hunk 1]
```diff
@@ -21,6 +21,17 @@ export default class Domains {
     for(appId) {
         return unpackData(this._client.apiClient().get(`/apps/${appId}/domains`), "domains")
     }
+
+    /**
+     * Link a CHIBRE name to a specific application
```
Comment (yannski) [inline src/Domains/index.js]: CHIBRE does not sound right :)

Comment (ichbinkour) [reply src/Domains/index.js]: Sorry, it was for debugging.

File: src/Domains/index.js [hunk 2]
```diff
@@ -21,6 +21,17 @@ export default class Domains {
     for(appId) {
         return unpackData(this._client.apiClient().get(`/apps/${appId}/domains`), "domains")
     }
+
+    /**
+     * Link a domain name to a specific application
+     * @see http://developers.scalingo.com/domains#link-a-domain-name-to-an-application
+     * @param {String} appId ID of the app to post to the specified application
+     * @param {DomainParams} domain An object of the domain to link
+     * @return {Promise<Domain[] | APIError>}
```
Comment (johnsudaar) [inline src/Domains/index.js]: The API returns a single Domain not an array of domain

File: src/Domains/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import {unpackData} from "../utils";
 
 /**
- * Domains API Client
+ * Containers API Client
```
Comment (yannski) [inline src/Domains/index.js]: Should be Domains, indeed

File: src/Domains/index.js [hunk 4]
```diff
@@ -40,9 +40,9 @@ export default class Domains {
  * @property {String} name Hostname your want to associate with the app
  * @property {String} tlscert Subject of the submitted certificate
  * @property {String} tlskey Private key type and length
- * @property {Boolean} ssl Flag if SSL with a custom certificate is enabled
+ * @property {String} ssl Flag if SSL with a custom certificate is enabled
  * @property {Date} validity Once a certificate has been submitted, display the validity of it
- * @property {Boolean} canonical The domain is the canonical domain of this application
+ * @property {String} canonical The domain is the canonical domain of this application
```
Comment (yannski) [inline src/Domains/index.js]: I do think "canonical" is a boolean...

File: src/Domains/index.js [hunk 5]
```diff
@@ -29,7 +40,15 @@ export default class Domains {
  * @property {String} name Hostname your want to associate with the app
  * @property {String} tlscert Subject of the submitted certificate
  * @property {String} tlskey Private key type and length
- * @property {Boolean} ssl Flag if SSL with a custom certificate is enabled
+ * @property {String} ssl Flag if SSL with a custom certificate is enabled
```
Comment (johnsudaar) [inline src/Domains/index.js]: ssl is a boolean too

**Pull request merged**